test/reshape.jl

```diff
@@ -396,6 +396,22 @@ end
     @test df_flat_cat.b isa CategoricalArray
 end
 
+@testset "flatten multiple columns" begin
+    df = DataFrame(a = [1, 2], b = [[1, 2], [3, 4]], c = [[5, 6], [7, 8]])
+    ref = DataFrame(a = [1, 1, 2, 2], b = [1, 2, 3, 4], c = [5, 6, 7, 8])
+    @test flatten(df, [:b, :c])        == ref
```

Comment:
e.g. here add the test `flatten(df, [:c, :b])`.

Also do not align `==` just use a single space